auction-deploy/tests/test_cli.py

```diff
@@ -66,6 +66,20 @@ def test_cli_start_auction(runner, deployed_auction_address):
     assert result.exit_code == 0
 
 
+def test_cli_start_auction_with_auto_nonce(
+    runner, deployed_auction_address, keystores, key_password
+):
+    """test the auto-nonce option. we only do this for the start-auction"""
+
+    result = runner.invoke(
+        main,
+        args=f"start-auction --jsonrpc test --keystore {keystores[0]} --auction-address {deployed_auction_address}",
```

Comment:
Does it make sense that there is no `auto-nonce` in the args here ?

Reply:
Right, `--auto-nonce` should have been there.

The print statement might provide useful information in case of an error. If the test succeeds, you won't see the output anyway.

I will open another PR as I've already merged this one.

Reply:
https://github.com/trustlines-protocol/blockchain/pull/59